### ISSUE TITLE
[$30][MSFT-160] Detail in prize should been read automatic or can go there by keyboard, but both can't.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2847]
**Steps to Reproduce**
1. Navigate to https://www.topcoder.com/challenges?
2. Go to My Challenges/open for registration . 
3. Try to go prize details form keyboard.

**Expected Result**
User should get the prize details and able to read it.

**Actual Result**
 When you go to prize with keyboard the detail of prize didn't show, and can't be read.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 2.1.1 Keyboard
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=ea73aab1%2D4440%2D4d96%2D93dc%2D8069c62e4283&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FWinner123%2Fbug4

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096049 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096049 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096049 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@veshu this has been already cover in PR of issue https://github.com/topcoder-platform/community-app/issues/2828

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096049 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2847 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097126 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097126 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by r0hit-gupta** [comment]
@PrakashDurlabhji @codeMinter this has been solved by PR https://github.com/topcoder-platform/community-app/pull/3157

**Comment by PrakashDurlabhji** [comment]
@veshu @codeMinter rohit has copied my PR and nothing else.you can check it.

these both PR  code are same: 
https://github.com/topcoder-platform/community-app/pull/2949/files
https://github.com/topcoder-platform/community-app/pull/3157/files

@codeMinter if his code will be considered than why not mine?

**Comment by lakshmiathreya** [comment]
Prize amount being read correctly, on hover and tab accessible. Verified on MacOS/Chrome/Voice Over & Win10. This is related to #2828 and #2889.

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```
